packages/grafana-data/src/utils/csv.ts

```diff
@@ -289,7 +290,7 @@ export function toCSV(data: DataFrame[], config?: CSVConfig): string {
         if (i > 0) {
           csv += config.delimiter;
         }
-        csv += fields[i].name;
+        csv += getFieldDisplayName(fields[i]);
```

Comment:
Not sure when to pass the dataFrame props but maybe @torkelo can explain further.
```suggestion
        csv += getFieldDisplayName(fields[i], series, data);
```

Reply:
You you should pass DataFrame here as the second argument, hmm not sure why DataFrame argument is optional :)

Reply:
but what does it do? I don't see any difference in the output?